### PR TITLE
AtoM on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by sbreker**
- Added etc/php7.4 folder (linked to PHP 7.2 folder)
- Added PHP packages installation for PHP 7.4
- Updated package python-pip to python3-pip

### CODE REVIEW

File: vars/Ubuntu-20.04.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+php_version: "{{ atom_php_version|default('7.4') }}"
```
Comment (djjuhasz) [inline vars/Ubuntu-20.04.yml:1]: It's not a blocker, but I would just hardcode the php_version.  I don't see any advantage to making php_version a variable, but hardcoding the version the package names.

File: templates/etc/php/7.4 [hunk 2]
```diff
@@ -0,0 +1 @@
+7.0
```
Comment (djjuhasz) [inline templates/etc/php/7.4:1]: I don't get this, does it just point the 7.4 install to use the `templates/etc/php/7.0/fpm/pool.d/atom.conf`?

Comment (sbreker) [reply templates/etc/php/7.4:1]: Yes - just a symlink. The PHP 7.2 version is doing the same thing.

File: vars/Ubuntu-20.04.yml (hunk 1 above)
Comment (sbreker) [reply vars/Ubuntu-20.04.yml]: Hi @djjuhasz - Good point! I have amended the commit to assign php_version to "7.4"

**Review by jraddaoui — APPROVED** [review]

File: tasks/deps.yml [hunk 3]
```diff
@@ -56,7 +56,24 @@
...
+    - "imagemagick"       # ↓ AtoM dependencies
+    - "ghostscript"       #
+    - "poppler-utils"     #
+    - "ffmpeg"            #
+    - "git"               # ↓ Build dependencies
+    - "npm"               #
+    - "make"              #
+    - "python3-pip"       #
+    - "python-setuptools" #
+    - "openjdk-8-jre-headless" # Needed by FOP
```
Comment (jraddaoui) [inline tasks/deps.yml:75]: Maybe add python-is-python3 in here.

Comment (sbreker) [reply tasks/deps.yml:76]: Added - see latest commit @jraddaoui

**Review by jraddaoui — APPROVED** [review]

**Pull request merged**